tests/test_plot.py

```diff
@@ -620,6 +620,6 @@ def pdf(x, a=1 / np.sqrt(2 * np.pi), x0=0, sigma=1, offset=0):
 
     fig, ax = plt.subplots()
```

Comment:
As there is still overlap of y-axes on the test plots, can this get changed to 
```suggestion
    https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.setp.html
```
? That then fixes the axis overlap problem
![debug_hist_1](https://user-images.githubusercontent.com/5142394/112247391-391e5d00-8c22-11eb-95f0-e117e3e2308b.png)

Reply:
...and now with me suggesting actual code and not a URL by accident. sigh.